features/zoeken-en-raadplegen/met-burgerservicenummer/fout-cases.feature

```diff
@@ -0,0 +1,107 @@
+#language: nl
+
+Functionaliteit: Raadpleeg met burgerservicenummer - fout cases
+
+Rule: Er moet minimaal één burgerservicenummer worden opgegeven
+
+  @fout-case
+  Scenario: De burgerservicenummer parameter is niet opgegeven
+    Als personen wordt gezocht met de volgende parameters
+    | naam   | waarde                          |
+    | type   | RaadpleegMetBurgerservicenummer |
+    | fields | burgerservicenummer             |
+    Dan heeft de response een object met de volgende gegevens
+    | naam     | waarde                                                      |
+    | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1 |
+    | title    | Minimale combinatie van parameters moet worden opgegeven.   |
+    | status   | 400                                                         |
+    | detail   | De foutieve parameter(s) zijn: burgerservicenummer.         |
+    | code     | paramsCombination                                           |
+    | instance | /haalcentraal/api/brp/personen                              |
+    En heeft het object de volgende 'invalidParams' gegevens
+    | code     | name                | reason                  |
```

Comment:
Waarom geef je hier maar ook in de andere scenario's van dit en andere fout-cases.feature bestanden niet ook het 'type' veld terug?

Reply:
Nee, ik bedoel dit type:

![image](https://user-images.githubusercontent.com/20185784/203797871-36e24fcf-3e4c-4adb-8ce0-f4f69244578b.png)

Reply:
dat kunnen we niet kwijt in de InvalidParams. De vraag is ook of het meerwaarde geeft om het type door te geven